templates/injector.template

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Injector generated by Zend\Di
+ */
+
+%namespace%
```

Comment:
Same comment here as for above about line between namespace and `use` declarations.